data/messages.json

```diff
@@ -519,6 +523,7 @@
   "RADIO_PREFERRED_DISABLED": "You have disabled the in-game radio.",
 
   "REPORT_MESSAGE": "{33AA33}Your report has been delivered to the crew{FFFFFF}: Player: %s (Id:%d) - Reason: %s",
+  "REPORT_ERROR_NPC": "@error You can't report an NPC!",
```

Comment:
NIT: a npc

"A non playing character"    "an" is used when the next word is a vowel (sound) "an apple" or the first letter isn't really spoken so still does: "an hour"